nextflow_schema.json

```diff
@@ -45,30 +45,75 @@
             }
         },
         "taxon_profiling_databases": {
-            "title": "Taxon-profiling Databases",
+            "title": "taxon_profiling_databases",
```

Comment:
Should this title be formatted for us or for the end-user?

Reply:
fixed, yeah I messed up the copy paste